arcgispro/3D Data Co-Registration.pyt

```diff
@@ -353,27 +353,88 @@ class Register_MultiType(object):
         }
         codem_run_config = codem.CodemRunConfig(fnd_full_path, aoi_full_path, **kwargs)
         config = dataclasses.asdict(codem_run_config)
+        
+        #Add output to details pane with parameters and their values
+        arcpy.AddMessage("=============PARAMETERS=============")
+        for print_parameter in config:
+            param_string = print_parameter + " = " + str(config[print_parameter])
```

Comment:
We can rename this instead of doing string concatenation like this, we can do 

```python
for parameter, value in config.items():
  arcpy.AddMessage(f"{print_parameter}={value}")
```